docs/python-reference/api.rst

```diff
@@ -46,6 +46,8 @@ General
 Comparison
 ~~~~~~~~~~
 
+Comparison operators compare element-wise and *ignore variances*.
+
```

Comment:
Is it worth saying something about this in release notes?

Reply:
It is not a breaking change I believe, and not a major feature either, so no? But we can discuss about whether we want more detailed release notes in the future.